tasks/setup.yml

```diff
@@ -0,0 +1,31 @@
+---
+
+- name: Gather os specific variables
+  include_vars: "{{ item }}"
+  with_first_found:
+    - files:
+        - "{{ ansible_distribution }}-{{ ansible_distribution_major_version}}.yml"
+        - "{{ ansible_distribution }}.yml"
+        - "{{ ansible_os_family }}.yml"
+      skip: true
+  tags: vars
```

Comment:
Probably nicer to do this in main.yml.

Reply:
I wasn't sure if we should make it so the playbooks can be run separately.

Reply:
Fair point.